plugins/statistics.koplugin/main.lua

```diff
@@ -486,6 +490,7 @@ function ReaderStatistics:saveSettings(fields)
 end
 
 function ReaderStatistics:onReadSettings(config)
+    -- delay initialization for accurate total pages count of the doc
```

Comment:
CrengineC will first report the total page count to 1 and then after post initialization process it will report the right count. So this schedule in will make the stats initialization after the document initialization. Considering all processings are blocked the delay time does not matter.